.github/workflows/on-pr-close.yml

```diff
@@ -41,7 +41,7 @@ jobs:
     steps:
       - uses: actions/checkout@v4
         with:
-          ref: ${{ env.BRANCH_NAME }}
+          ref: ${{ env.BRANCH_NAME || 'main' }}
```

Comment:
So because the branch gets deleted it needs to run the build in "main". If it's just a regular close of PR this will remain on the branch name.